modules/nf-core/tcoffee/irmsd/main.nf

```diff
@@ -24,14 +24,16 @@ process TCOFFEE_IRMSD {
     """
     export TEMP='./'
 
+    unpigz -cdf $msa > msafile &
```

Comment:
Here as well something like [this](https://github.com/nf-core/modules/blob/8fc1d24c710ebe1d5de0f2447ec9439fd3d9d66a/modules/nf-core/abritamr/run/main.nf#L30-L32) would be better in my opinion